plone/app/layout/viewlets/content.py

```diff
@@ -71,43 +71,7 @@ def show(self):
             ISecuritySchema,
             prefix='plone',
         )
-        return not self.anonymous or settings.allow_anon_views_about
```

Comment:
@thet well, another proof that the toolbar design was half baked and the removal of the viewlet was a bad idea in the first place; I would prefer to leave the document byline visible for all, but in the past we had some heated discussions because of that as some people simply wanted to move away from it without thinking on all the unintended consequences.

IMO the document byline helps to remove the cluttering of the toolbar and there are many regressions because of its removal.

@davilima6 the clock icon is not solving all cases; see https://github.com/plone/Products.CMFPlone/issues/1575